tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -1,18 +1,28 @@
 import pytest
+import unittest
 
-from coalib.bearlib.aspects import AspectTypeError as aspectTypeError
+from coalib.bearlib.aspects import (
+    AspectNotFoundError, AspectTypeError as aspectTypeError)
 from coalib.bearlib.aspects.collections import AspectList
 from coalib.bearlib.aspects.meta import isaspect
 from coalib.bearlib.aspects.Metadata import Metadata
 
 
-class AspectListTest:
+class AspectListTest(unittest.TestCase):
 
     def test__init__(self):
-        with pytest.raises(aspectTypeError) as exc:
+        list_of_aspect = AspectList(['CommitMessage.Shortlog',
+                                     'CommitMessage.Body'])
+        mix_of_aspect = AspectList(['CommitMessage.Shortlog',
+                                    Metadata.CommitMessage.Body])
+        assert isinstance(list_of_aspect, AspectList)
```

Comment:
replace `assert` with `self.assert...(...)` (This was mentioned to you in another PR just merged; fix all of your PRs)